shared/django_apps/dummy_settings.py

```diff
@@ -27,15 +27,15 @@
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.postgresql",
-        "NAME": "POSTGRES_USER",
+        "NAME": "postgres",
         "USER": "postgres",
         "PASSWORD": "password",
         "HOST": "postgres",
         "PORT": 5432,
     },
     "timeseries": {
         "ENGINE": "django.db.backends.postgresql",
-        "NAME": "POSTGRES_USER",
+        "NAME": "postgres",
```

Comment:
this was forgotten in https://github.com/codecov/shared/commit/9c1a9da61a778b1e04c89503df501a507c7339c2